docs/resources/resourcemanager_project.md

```diff
@@ -35,9 +35,20 @@ resource "stackit_resourcemanager_project" "example" {
 ### Optional
 
 - `labels` (Map of String) Labels are key-value string pairs which can be attached to a resource container. A label key must match the regex [A-ZÄÜÖa-zäüöß0-9_-]{1,64}. A label value must match the regex ^$|[A-ZÄÜÖa-zäüöß0-9_-]{1,64}
+- `members` (Attributes List, Deprecated) The members assigned to the project. At least one subject needs to be a user, and not a client or service account. This value is only considered during creation. Changing it afterwards will have no effect.
```

Comment:
same here, this file should not be updated

Reply:
This change was created by the doc generation. I'll inspect how this happened

Reply:
I is expected, but we should delete it manually. Like described here: https://github.com/stackitcloud/terraform-provider-stackit/pull/611#discussion_r1887169437

Reply:
As discussed, unrelated change caused by missing doc generation on main, but will be kept to cleanup main as well.

Reply:
removed members section